### DISCUSSION TITLE
v1.3.3

### DISCUSSION THREAD

**Discussion by caarlos0** [caarlos0/fastcom-exporter#9] [Announcements]
## Changelog

4cecc5c fix: typo causing panic

## Docker images

- `docker pull ghcr.io/caarlos0/fastcom-exporter:v1.3.3`
- `docker pull caarlos0/fastcom-exporter:v1.3.3`
---

_Released with [GoReleaser Pro](https://goreleaser.com/pro)!_

<hr /><em>This discussion was created from the release <a href='https://github.com/caarlos0/fastcom-exporter/releases/tag/v1.3.3'>v1.3.3</a>.</em>

**Comment by Sysadminfromhell** [comment]
Hi there, is there any way to get the new statistics from the api?
there is a advanced part of the fast.com page which allows you to see your Upload, Jitter and Ping.
This Exporter doesnt support it yet, is it planned?

**Reply by caarlos0** [reply]
no plans for now, happy to review a PR though.
